src/sentry/sentry_apps/services/hook/impl.py

```diff
@@ -21,13 +21,61 @@ def update_webhook_and_events(
         with transaction.atomic(router.db_for_write(ServiceHook)):
             hooks = ServiceHook.objects.filter(application_id=application_id)
             if webhook_url:
-                for hook in hooks:
-                    hook.url = webhook_url
-                    hook.events = expand_events(events)
-                    hook.save()
-                return [serialize_service_hook(h) for h in hooks]
+                expanded_events = expand_events(events)
+                hooks.update(url=webhook_url, events=expanded_events)
             else:
                 deletions.exec_sync_many(list(hooks))
+            return [serialize_service_hook(h) for h in hooks]
+
+    def update_webhook_and_events_for_app_by_region(
```

Comment:
I want to do an rpc cutover from the `organization_id` routed `update_webhook_and_events` above to a region routed one. That's because when updating all webhooks for a sentry app, it's much clearer to aggregate all the regions a sentry app is installed in versus iterating through all installations/organizations to find the first one that exists in each region.

The cutover in PR steps -
PR 1 - add new RPC with region routing (this one)
PR 2 -  cutover all callsites to use this new RPC
PR 3 - remove the old function